sherpa/fit.py

```diff
@@ -572,18 +646,40 @@ def primini(self, statfunc, pars, parmins, parmaxes, statargs=(),
 
         This is a chi-square statistic where the variance is computed
         from model amplitudes derived in the previous iteration of the
-        fit. This 'Iterative Weighting' ([1]_) attempts to remove
+        fit. This 'Iterative Weighting' [1]_ attempts to remove
         biased estimates of model parameters which is inherent in
-        chi-square2 statistics ([2]_).
+        chi-square statistics [2]_.
+
+        Raises
+        ------
+        sherpa.utils.err.FitErr
+            This exception is raised if the statistic is not
+            supported. This method can only be used with
+            Chi-Square statistics with errors.
+
+        Notes
+        -----
+        The following keys are looked for in the ``itermethod_opts``
+        dictionary:
+
+        ========  =========  ===========
+        Key       Type       Description
+        ========  =========  ===========
+        tol       number     The tolerance used when comparing the statistic
+                             values.
+        maxiters  int        The maximum number of iterations.
+        ========  =========  ===========
 
         The variance in bin i is estimated to be:
 
-        sigma^2_i^j = S(i, t_s^(j-1)) + (A_s/A_b)^2 B_off(i, t_b^(j-1))
+        .. math::
+
+           \sigma(j)_i^2 = S(i, t_s(j-1)) + (A_s/A_b)^2 B_{off}(i, t_b(j-1))
```

Comment:
There's a number of different ways it could be done (including leaving it as it was), and I'm not sure what's best. So this discussion is good, if it comes to a conclusion...